docs/src/evaluation_metrics.md

```diff
@@ -0,0 +1,28 @@
+# Evaluation metrics
```

Comment:
I wonder if these should be docstrings? The reason I didn't do so from the start is that I meant this page to be more pedagogical (and hopefully to have references), instead of just documenting the programmatic API.